models/MerchandiseCollectionModel.ts

```diff
@@ -10,6 +10,9 @@ export class MerchandiseCollectionModel extends BaseEntity {
   @Column('varchar', { length: 255 })
   title: string;
 
+  @CreateDateColumn()
+  createdAt: Date;
+
```

Comment:
Regarding the `timestamptz` type, you can pass in a `type` property to the `@CreateDateColumn()` decorator as `timestamptz` to explicitly set the type appropriately.
```ts
  @CreateDateColumn({ type: 'timestamptz' })
  createdAt: Date;
```